src/components/SponsorTimeEditComponent.tsx

```diff
@@ -240,6 +247,25 @@ class SponsorTimeEditComponent extends React.Component<SponsorTimeEditProps, Spo
         );
     }
 
+    changeTimesWhenScrolling(index: number, e: React.WheelEvent, sponsorTime: SponsorTime): void {
+        const step = 0.01;
+        const sponsorTimeEdits = this.state.sponsorTimeEdits;
+        let timeAsNumber = utils.getFormattedTimeToSeconds(this.state.sponsorTimeEdits[index]);
+        if (timeAsNumber !== null && e.deltaY != 0) {
+            if (e.deltaY < 0){
+                timeAsNumber += step;
+            }else if (timeAsNumber >= step){
+                timeAsNumber -= step;
+            } else {
+                timeAsNumber = 0;
+            }
```

Comment:
if it is zero, ut just doesn't need to be changed, it doesn't have to be reset to zero.

Reply:
This more for if the time left is 0.003 seconds. It would stay without the  the else.

Reply:
`timeAsNumber >= step` isn't needed actually, because `utils.getFormattedTimeToSeconds` only return `>=0`, but if this were to be changed, it wouldnt be nice if this piece of code broke because of it.